sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -893,6 +893,16 @@ object SQLConf {
       .booleanConf
       .createWithDefault(false)
 
+  val THRIFTSERVER_QUERY_TIMEOUT =
+    buildConf("spark.sql.thriftServer.queryTimeout")
```

Comment:
Can you rewrite this as:
```
Set a query duration timeout in seconds in ThriftServer. If the timeout is set to a positive value, a running query will be cancelled automatically when the timeout is exceeded, otherwise the query continues to run till completion. Timeout values that are set for each statement via`java.sql.Statement.setQueryTimeout` take precedence.
```

Reply:
Updated.